tools/ops/policylambda.py

```diff
@@ -54,36 +56,81 @@ def render(p):
     if key_arn:
         properties['KmsKeyArn']
 
-    # Event render
-    revents = {}
-    if p.execution_mode == 'periodic':
-        revents = {
-            'PolicySchedule': {
-                'Type': 'Schedule',
-                'Properties': {
-                    'Schedule': p.data.get('mode', {}).get('schedule')}}
-        }
-    else:
-        events = [e for e in policy_lambda.get_events(None)
-                  if isinstance(e, mu.CloudWatchEventSource)]
-        if not events:
-            return
-
+    if p.execution_mode == 'periodic' or p.execution_mode == 'cloudtrail':
+        # Event render
         revents = {}
-        for idx, e in enumerate(events):
-            revents[
-                'PolicyTrigger%s' % string.ascii_uppercase[idx]] = {
-                    'Type': 'CloudWatchEvent',
+        if p.execution_mode == 'periodic':
+            revents = {
+                'PolicySchedule': {
+                    'Type': 'Schedule',
                     'Properties': {
-                        'Pattern': json.loads(e.render_event_pattern())}
+                        'Schedule': p.data.get('mode', {}).get('schedule')}}
             }
+        else:
```

Comment:
thanks for the followup pr, this block should be dedented to an if == 'cloudtrail' with the additional clause.

Reply:
updated based on feedback